### ISSUE TITLE
html5 havent info who unpaused game

### ISSUE THREAD

**Issue by Wenomka** [haxball/haxball-issues#175]
In new update we cant knows who unpaused games

**Comment by Simashax** [comment]
I agree, it needs to be implemented. It would be also useful if there was information about who and whom gave/removed admin rights

**Comment by Leo-MapCreator** [comment]
Yeah, im my opinion haxball is growing because it is exposing some actions that stayed hidden from the players and that are useful, like who kicked/banned who, who paused/unpaused the game, who gave/removed admin to whom, who stopped the game
